boards/mega-xplained/include/board.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ *               2016 Laurent Navet <laurent.navet@gmail.com>
+ *               2018 Matthew Blue <matthew.blue.neuro@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/*
+ * @defgroup    boards_mega-xplained
```

Comment:
Better moving the `@defgroup` and `@ingroup` in a separate doc.txt file. See #8516 and arduino-mkrfox1200 for examples.